rules/S6829/java/metadata.json

```
@@ -0,0 +1,25 @@
+{
+  "title": "@Autowired should be used when multiple constructors are provided",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "5min"
+  },
+  "tags": [
+    "spring"
+  ],
+  "defaultSeverity": "Minor",
+  "ruleSpecification": "RSPEC-6829",
+  "sqKey": "S6829",
+  "scope": "All",
+  "defaultQualityProfiles": ["Sonar way"],
+  "quickfix": "unfeasible",
```

Good thing the CI catches this kind of errors
```suggestion
  "quickfix": "infeasible",
```